visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -173,14 +172,6 @@ describe("codeMapMouseEventService", () => {
 	}
 
 	describe("constructor", () => {
-		it("should subscribe to hoverEvents", () => {
```

Comment:
Besides that we don't subscribe to it anymore, this kind of testing provides no value. Which user feature cares if something subscribes to something in its constructor? :/